google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -22,7 +22,7 @@ appengine.cloudsdk.location.label=Cloud SDK directory\:
 appengine.cloudsdk.location.browse.window.title=Browse to Cloud SDK directory
 appengine.cloudsdk.location.missing.message=Please select a Cloud SDK home directory.
 appengine.cloudsdk.project.missing.message=Please select a project.
-appengine.appyaml.location.browse.window.title=Browse for Your App.Yaml Location
+appengine.appyaml.location.browse.window.title=Browse for Your app.yaml Location
```

Comment:
maybe its must me, but browse "for" your app.yaml location sounds awkward. Maybe "to" instead.